_includes/svg/icon-quotes.liquid

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" class="svg-icon-quotes {{ include.class }}" alt="{{ include.alt }}">
```

Comment:
🤔 is this svg `icon-quotes`(in plural) different to `icon-quote` (in singular)? 
If it's duplicate we should remove on of them 😄

Reply:
😬 it's a duplicate, yes

Reply:
Ok, no problem!

Thanks